region/norfair/east.json

```diff
@@ -1838,10 +1838,15 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                    {"heatFrames": 120},
-                    "h_canHeatedBlueGateGlitch"
+                    {"heatFrames": 220},
+                    "canGateGlitch",
+                    {"or": [
+                      {"ammo": { "type": "Missile", "count": 1 }},
+                      {"ammo": { "type": "Super", "count": 1 }}
+                    ]}
                   ],
-                  "clearsObstacles": ["A"]
+                  "clearsObstacles": ["A"],
+                  "devNote": "Omitting gate glitch leniency, since the nearby Gamet farm allows refilling energy and ammo."
```

Comment:
This is interesting. The farm is good, but the room is heated. This will be the only gate glitch in logic if you only have a small number of ammo, which can be a bit confusing. It also only works because its in the same room - if there is a great farm in a neighboring room it won't be in logic. 

Something to compare to, should the green hill zone gate be in logic if the right door can be used to reset room? those hoppers can be farmed for missiles.

Reply:
I would be fine going either way on this; we could keep the leniency here so that the ammo requirements stay consistent across all gates. Or we could add more strats with reduced leniency based on good neighboring farms.

The Green Hill Zone sidehoppers have 34% Missile drop rate (60% with full everything else), so with having to reset the room every time they're not a great farm but also not bad. Neighboring good farms would be in the realm of possibility of something we could model with the new style of cross-room strats. There's still a delay from having to reset the room, but if you don't have Varia or tanks that still might be easier than in-room farming in Upper Norfair Farming Room. And like you said, there's a question of whether all of this makes it harder for the player to understand how gate glitch leniency works.

Reply:
The UN farm does become harder to follow the logic if you are basing it on GGG leniency value, which is what I've been doing.  Looking at it as something you could do, as a `X Difficulty` runner, then you would count the farm as an extension of equipment capacity.  I think I would rather ammo requirements stay consistent.  

GGG does feel a lot better with a nearby farm spot.  But that's better off just being a bonus.  Reloading rooms to farm is still going to feel bad, and most of the random enemy farm spots are still pretty slow as ammo per reload.

Including the farm spot here; 
Expert - no Etanks, 1 missile pack is fine
VH - 1 tank, 1 pack.  This gate glitch comes up in other randomizers and its pretty uncomfortable to attempt the gate and then refill on no tanks.
Hard - 1 tank for the room, and 2 missile packs seems ok.  I expect a lot of missed shots at hard.

Reply:
Makes sense, I'll put the strat back how it was and we could think about it more in a separate issue or PR.

For this strat I think the ammo requirement is less of an issue than the energy requirement. On Hard, the 14 leniency shots are 60 heat frames each, plus the base 100 heat frames for heated gate glitch on top of 120 heat frames in the strat. Adds up to 1060 heat frames. With the Hard multiplier, that becomes 1590 heat frames, or 397 energy. So it would need 3 tanks, or 4 tanks if you need to be able to make it back to the farm after opening the gate. 

Agree that 1 tank seems like enough for this to be in logic on Hard. And with only 1 tank, I'm not sure that a second missile pack really makes the strat easier. Probably 5 attempts is enough that you would want to be going back to farm energy anyway by that point, and this will refill your Missiles. The way I wrote though it would put it in logic with no tanks, which seems unforgiving for Hard (and maybe Very Hard)